src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -9129,9 +9129,40 @@ void M()
 }", "C");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task EscapedAwaitIdentifierInAsyncMethod()
+        {
+            var markup = @"
+class await { }
+
+class C
+{
+    async void Foo()
+    {
+        a$$
+    }
+}";
+            await VerifyItemExistsAsync(markup, "@await");
```

Comment:
this test should probably also have `await VerifyItemIsAbsentAsync(markup, "await");`